### PR TITLE
Add subscriptions to permissions page

### PR DESCRIPTION
**Pull request by johughes1204**

### CODE REVIEW

File: docs/authentication/Tokens/permissions.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ The following tables show a breakdown of the API endpoint actions that are avail
  | `/settings/account-authentication-settings`  | ⛔️ | ⛔️ |
  | `/settings/customer-authentication-settings` | ⛔️ | ⛔️ |
  | `/settings/extensions`                       | ⛔️ | ⛔️ |
+| `subscriptions` | |
```
Comment (cjmarkham) [inline docs/authentication/Tokens/permissions.md:62]:
```suggestion
| `/subscriptions` | ⛔️ | ⛔️ |
```

File: docs/authentication/Tokens/permissions.md [hunk 2]
```diff
@@ -99,6 +100,7 @@ The following tables show a breakdown of the API endpoint actions that are avail
  | `/settings/account-authentication-settings`  | ⛔️ | ⛔️ |
  | `/settings/customer-authentication-settings` | ⛔️ | ⛔️ |
  | `/settings/extensions`                       | ⛔️ | ⛔️ |
+ | `subscriptions` | |
```
Comment (cjmarkham) [inline docs/authentication/Tokens/permissions.md:103]:
```suggestion
 | `/subscriptions` | ⛔️ | ⛔️ |
```

File: docs/authentication/Tokens/permissions.md [hunk 3]
```diff
@@ -139,6 +141,7 @@ The following tables show a breakdown of the API endpoint actions that are avail
 | `/settings/account-authentication-settings`  | ⛔️ | ⛔️ |
 | `/settings/customer-authentication-settings` | ⛔️ | ⛔️ |
 | `/settings/extensions`                       | ⛔️ | ⛔️ |
+| `subscriptions` | |
```
Comment (cjmarkham) [inline docs/authentication/Tokens/permissions.md:144]:
```suggestion
| `/subscriptions` | ⛔️ | ⛔️ |
```

**Comment by johughes1204** [comment]
Closing this as these changes have already been added. I also have a ticket to add more information about authentication in subscriptions. https://elasticpath.atlassian.net/jira/software/c/projects/DB/boards/388?selectedIssue=DB-1060

**Pull request closed without merging**